internal/config/nsmgr/nsmgr_linux.go

```diff
@@ -132,6 +132,10 @@ func (mgr *NamespaceManager) NewPodNamespaces(cfg *PodNamespacesConfig) ([]Names
 		return nil, fmt.Errorf("failed to pin namespaces %v: %s %w", cfg.Namespaces, output, err)
 	}
 
+	if len(output) != 0 {
+		logrus.Debugf("Got output from pinns: %s", output)
+	}
```

Comment:
No need to wrap debug log with a check, you probably want to see any value there, if debug level is set.

```suggestion
	logrus.Debugf("Got output from pinns: %v", output)
```